src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{
+
+    @Override
+    public void doWork() {
+        System.out.println("the bulldozer started it`s work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("the bulldozer stopped it`s work");
```

Comment:
The message should be more informative and indicate that it is a Bulldozer that has stopped working. For example: 'Bulldozer stopped its work.' Also, use the correct possessive form 'its' instead of 'it`s'.